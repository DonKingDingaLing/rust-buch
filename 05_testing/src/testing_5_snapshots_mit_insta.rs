#[test]
fn test_hello_world() {
    insta::assert_snapshot!(
        "Ich bin ein Snapshot, wenn auch \
    nur ein kleiner!"
    );
}
