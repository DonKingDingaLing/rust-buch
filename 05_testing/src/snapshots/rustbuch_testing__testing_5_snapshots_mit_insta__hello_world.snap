---
source: src/testing_5_snapshots_mit_insta.rs
expression: "\"Ich bin ein Snapshot, wenn auch \\\n    nur ein kleiner!\""

---
Ich bin ein Snapshot, wenn auch nur ein kleiner!
